src/app/components/Byline/index.test.tsx

```diff
@@ -75,9 +75,9 @@ describe('Byline', () => {
   });
 
   it('should correctly use the buildIChefURL function to create the image url', () => {
-    render(<Byline blocks={bylineWithPngPhoto} />);
+    const { container } = render(<Byline blocks={bylineWithPngPhoto} />);
 
-    const imageSrc = screen.getAllByRole('img');
```

Comment:
This could also have been fixed by using `screen.getAllByRole('presentation');` - but shall we just use your fix?

Reply:
Oh interesting. Maybe thats better actually?

Reply:
I'll change it to that, feels more accurate.